projects/fal/tests/test_stability.py

```diff
@@ -11,11 +11,8 @@
 PACKAGE_NAME = "fal"
 
 
+@pytest.mark.xfail(reason="after migrating to cloudpickle we get a serialization error")
```

Comment:
We now get a serilazation error about stuff like thread locks and other stuff before we can actually send it off to the server and see that it fails to deserialize. We should probably start doing proactive analysis of the user functions before we send them off instead of trying to react to server errors.